### PR TITLE
refactor card styles

### PR DESCRIPTION
**Pull request by ceciliamvrie**
Modified: 
• Card

I added the "hover" styles to match the spec whenever 
someone hovers their mouse over the card component.
I made sure that it only effects the cards with the hover 
property in them. 

Issues: none

### CODE REVIEW

**Comment by ceciliamvrie** [comment]
Here's the link! => https://docs-plvabcvcyr.now.sh

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "styled-material-components",
-  "version": "0.0.33",
+  "version": "0.0.36",
```
Comment (ashwoodall) [inline package.json]: Any reason for not bumping to 0.0.34?

Comment (ceciliamvrie) [reply package.json]: Hmm I believe that the latest version is .35, and so I was bumping it to .36

Comment (martahj) [reply package.json]: Looks like this [other open PR](https://github.com/ConciergeAuctions/styled-material-components/pull/139/files) bumps the package to .35 (so will need to be merged before this one).. don't know what happened to .34 though 🤔

**Comment by brad-decker** [comment]
:+1: 💯

**Pull request merged**